lib/chord.js

```diff
@@ -67,15 +71,43 @@ class Chord {
   transpose(amount) {
     return new Chord(transpose(this.root, amount), this.quality, transpose(this.bass, amount));
   }
+}
 
-  static isChord(str) {
-    return str.match(chordPattern) ? true : false;
+const compareNotes = (note1 = '', note2 = '') => NOTE_SEQUENCE.get(note1) - NOTE_SEQUENCE.get(note2);
+
+function compareQualities(quality1 = '', quality2 = '') {
+  if (quality1.length != quality2.length) {
+    return quality1.length - quality2.length;
   }
+  return quality1.localeCompare(quality2);
+};
 
-  static parse(str) {
-    const tokens = str.match(chordPattern);
-    return new Chord(tokens[1], tokens[2], tokens[3]);
+function compareChords(chord1, chord2) {
+  let cmp = compareNotes(chord1.root, chord2.root);
+  if (cmp != 0) {
+    return cmp;
+  }
+  cmp = compareQualities(chord1.quality, chord2.quality);
+  if (cmp != 0) {
+    return cmp;
   }
+  return compareNotes(chord1.bass, chord2.bass);
+};
+
+function isChord(str) {
+  return str.match(CHORD_PATTERN) ? true : false;
```

Comment:
This could be simplified to `return !!str.match(CHORD_PATTERN);`

Reply:
It's nice having someone who actually knows and likes Javascript.